src/app/core/data/entries-response-parsing.service.ts

```diff
@@ -0,0 +1,42 @@
+import { isNotEmpty } from '../../shared/empty.util';
+import { ObjectCacheService } from '../cache/object-cache.service';
+import { ErrorResponse, GenericSuccessResponse, RestResponse } from '../cache/response.models';
+import { DSpaceRESTV2Response } from '../dspace-rest-v2/dspace-rest-v2-response.model';
+import { DSpaceSerializer } from '../dspace-rest-v2/dspace.serializer';
+import { BaseResponseParsingService } from './base-response-parsing.service';
+import { ResponseParsingService } from './parsing.service';
+import { RestRequest } from './request.models';
+import { CacheableObject } from '../cache/object-cache.reducer';
+import { GenericConstructor } from '../shared/generic-constructor';
+
+export abstract class EntriesResponseParsingService<T extends CacheableObject> extends BaseResponseParsingService implements ResponseParsingService {
```

Comment:
Please add TypeDocs to this new Service & it's `parse()` method below

Reply:
done